### ISSUE TITLE
What is the learning rate for efficient net?

### ISSUE THREAD

**Issue by CoinCheung** [tensorflow/tpu#509]
Hi, 

I read from the code that the learning rate of efficient net is 0.016 when the batchsize is 256. And it grows to 0.128 when the batch size is 2048 according to the linear rule. However, in the paper, the learning is 0.256. Is the experiment in the paper use batchsize of 4096 ?

**Comment by saberkun** [comment]
Yes. 2048 is to fit TPUv2.
We primarily use 4096 with TPUv3.

**Comment by mingxingtan** [comment]
Yep, better to use batch size 4096 if using TPUv3 or as long as it doesn't run OOM.
